iozone/iozone_run.sh

```diff
@@ -973,7 +973,27 @@ execute_iozone()
                         test_specific_args="-I -n ${page_size}k -g ${dio_maxfile}m -y 64k -q 1m -i 0 -i 1 -i 2 -i 3 -i 4 -i 5"
                         do_test "Direct_IO" "directio" ${test_specific_args}
                 fi
-        else
+
+                if [[ ${do_out_of_cache} -eq 1 ]]; then
+                        test_specific_args="-n ${out_of_cache_start}m -g ${out_of_cache_end}m -y 8k -q 1m"
+                        if [[ ${do_eat_mem} == 1 ]];then
+                               ${eatmem_exe} ${memory_to_take} &
+                               PID=$!
+                               sleep 180
```

Comment:
We should figure out a way to tell us when the memory is consumed.  It is possible that we could start running when we are still consuming memory.